languages/python/exercises/concept/tuples/.docs/after.md

```diff
@@ -0,0 +1,145 @@
+In Python, a tuple is an immutable collection of items in _sequence_. Like most collections, tuples can hold any (or multiple) data type(s) -- including other tuples. Like any sequence, items are referenced by 0-based index number. Tuples support all [common sequence operations](https://docs.python.org/3/library/stdtypes.html#common-sequence-operations).
```

Comment:
Is there a resource to link to for the `tuple` concept?

Reply:
Added in `https://github.com/exercism/v3/blob/master/languages/python/reference/concepts/builtin_types/tuple.md`, which needs additional info.  In particular, the whole _homogenous_ vs _heterogeneous_  **data** (as opposed to data _**type**_) is very confusing to most people.  I have a note to add clarification links to docs and articles there - or generally more explanation.